tests/derive_iden_tests.rs

```diff
@@ -0,0 +1,37 @@
+pub mod common;
+pub use common::{features::*, setup::*, TestContext};
+use sea_orm::entity::prelude::*;
+use sea_orm_macros::DeriveIden;
+
+#[derive(DeriveIden)]
+pub enum Class {
+    Id,
+    Title,
+    Text,
+}
+
+#[derive(DeriveIden)]
+struct Glyph;
```

Comment:
Can we also test renaming the iden of unit struct?
```suggestion
#[derive(DeriveIden)]
#[sea_orm(iden = "_glyph")]
struct Glyph;
```

Reply:
We should create a new struct to test it